COOKBOOK-FIXERS.md

```diff
@@ -309,11 +312,11 @@ class RemoveCommentsFixer extends AbstractFixer
         $tokens = Tokens::fromCode($content);
 
         $foundComments = $tokens->findGivenKind(T_COMMENT);
-        foreach($foundComments as $index => $token){
+        foreach ($foundComments as $index => $token) {
```

Comment:
I was not around computer for few days, now I am back. @keradus yeah, you are right - that's the idea. However, @stof did now exactly the same thing you did when you first reviewed the COOKBOOK. So may be, after it will pacify more people formatting everything than trying to make a point in the documentation.